docs/pages/billing/faq.mdx

```diff
@@ -112,4 +112,8 @@ To view the cost of a large build, go to [Billing](https://expo.dev/settings/bil
 
 ### How do I increase build concurrencies on my account?
 
-[Contact us](https://expo.dev/contact) for more information.
+If you're already subscribed to a paid EAS plan, you can buy additional concurrencies in [the Add-ons section on the Billing](https://expo.dev/settings/billing#addons) page.
+
+If you're still on the Free plan, you will need to set up a paid subscription. [Click here](https://expo.dev/accounts/[account]/settings/billing/cart) to choose a plan. You can select a number of additional concurrencies to add to your subscription on the checkout page.
```

Comment:
```suggestion
If you're on the Free plan, you'll need to set up a paid subscription. [Click here](https://expo.dev/accounts/[account]/settings/billing/cart) to choose a new plan. Then, select a number of additional concurrencies to add to your subscription on the checkout page.
```

"If you're still" sounds like a bit aggressive to me. I think we remove the word "still" here.